src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -318,6 +323,9 @@ public InferenceBackendType Backend
         /// </remarks>
         /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
         /// <since_tizen> 6 </since_tizen>
+        /// <remarks>
+        /// Deprecated.(API Level 8) Not used.
```

Comment:
Please use Obsolete attribute instead of <remarks> tag.
```suggestion
    [Obsolete("Please do not use! This will be deprecated in level 8.")]
```

Reply:
Used Obsolete tag.